Stylesheet/css/general.css

```diff
@@ -3,15 +3,7 @@ html, body {
     margin: 0;
     padding: 0;
     font-size: 100%;
-}
-
-.img-responsive {
-    max-width: 100%;
-    height: auto;
-}
-
-.img-round {
-    border-radius: 50%;
+    font-family: var(--st-subheader-text);
 }
 
 .img-square {
```

Comment:
Please check this class .img-square syntax

Reply:
Yes checked, done it, thank you